viewer.js

```diff
@@ -64,7 +66,15 @@ function getNodeLabel(node) {
         continue;
       }
       const valueStr = typeof value === 'number' ? value.toFixed(6).replace(/\.?0*$/, '') : value;
-      label += `\n${key}: ${valueStr}`;
+        label += `\n${key}: ${valueStr}`;
+        if(key == "MessageId") {
```

Comment:
```suggestion
        if (key === 'MessageId') {
```
(please use single quotes so the coding style is more consistent)

Reply:
Ok